### DISCUSSION TITLE
Stability

### DISCUSSION THREAD

**Discussion by mohelsayedplus** [AxxAxx/AxxSolder#31] [General]
Is this project stable enough yet to build or shall i wait for another hw changes ?

**Comment by AxxAxx** [comment]
Hi! 
The current HW works. But I'm working on a new hardware that will be released in ~1-2 month. This HW release will have new pin assignment to the MCU and thus new firmware will not automatically be backwards compatible to (todays) hardware.

The new hardware will introduce: 
- USB-C PD
- A new opamp that will allow for better current measurement which in turn will allow for better temperature control (feed forward control) and sensing if the cartridge is installed or not by sending a short voltage pulse and read the current drawn
- The temperature measurement saturates at 440 deg C for C245 cartridges in the current HW. Therefore the max allowed temperature for C245 cartridges is 430 deg C for now. This will also be fixed. 

So if you have time I would advise you to wait for this upcoming revision. But if you do not need the above features you can go for the hardware availible today.

**Reply by AxxAxx** [reply]
AxxSolder 3.0 is now released!
